augur/filter.py

```diff
@@ -1748,6 +1752,7 @@ def calculate_sequences_per_group(target_max_value, counts_per_group, probabilis
         )
     except TooManyGroupsError as error:
         if probabilistic:
+            print(f"WARNING: {error}")
```

Comment:
This is a nice reuse of the original error message! Regarding my other comment, if we change the API of `calculate_sequences_per_group` to also return a boolean reflecting whether probabilistic sampling happened or not, we could push this warning message up to the `run` function (where we'll have a lack of exception but a `True` value for the `is_probabilistic` return value).

Reply:
Fixed: https://github.com/nextstrain/augur/commit/6784697a16e16ffc35a995e5aadf873acda8fe06